tests/framework/microvm.py

```diff
@@ -704,13 +717,28 @@ def add_drive(
     def add_vhost_user_drive(
         self,
         drive_id,
-        socket,
+        host_mem_path,
```

Comment:
nit: shouldn't this be just a `file_path`/`disk_path`? `host_mem_path` reminds me of the `memfd`.

Reply:
updated `path_on_host` like in `add_drive()`